### PR TITLE
[CognitiveServices] LUIS Programmatic API Spec

### PR DESCRIPTION
**Pull request by pcostantini**
This PR includes the API Spec for the LUIS Runtime endpoint (Predictions):
https://westus.dev.cognitive.microsoft.com/docs/services/5890b47c39e2bb17b84a55ff/operations/5890b47c39e2bb052c5b9c2f

The whole the spec has been reviewed and updated.

----

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json [hunk 1]
```diff
@@ -0,0 +1,6225 @@
...
+          },
+          {
+            "$ref": "#/parameters/TakeInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If there's no trained model, nothing is returned in the response. If there's a trained model and the active learning algorithm finds any relevant queries, they are returned with the model predictions. If there's a trained model but the active learning algorithm didn't find any relevant queries, an empty list is returned in the response.",
+            "schema": {
+
```
Comment (alvadb) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json]: Please add description to properties.

Comment (alvadb) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json]: Please add description for properties.

Comment (alvadb) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json]: Please add description to properties.

Comment (alvadb) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json]: Please add description to properties.

**Review by alvadb — CHANGES_REQUESTED** [review]

**Comment by alvadb** [comment]
Please review and fix linter and model errors:
linter: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/313223806
model: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/313223807

**Comment by enzocanoo** [comment]
Hi @alvadb 

We reviewed the lint errors and fixed almost all of them.
The followings autorest-lint errors are related to the current implementation of LUIS API and cannot be addressed without **introducing breaking changes to the current live** API:

- Delete operations with request body (DeleteMustNotHaveRequestBody error)
  Both operations use the request body to specify the entity to delete

- Properties not using camelCase style (DefinitionsPropertiesNamesCamelCase error)
  All these properties are defined as-is from the LUIS API and cannot be changed in the Swagger without updating the API implementation

Regards,
-EnzoC.

**Review by alvadb — APPROVED** [review]

**Pull request merged**